cpp-package/example/lenet.cpp

```diff
@@ -169,7 +161,9 @@ class Lenet {
         }
       }
 
-      LG << "Iter " << ITER
+      auto e_end = std::chrono::high_resolution_clock::now();
+      auto time_taken = std::chrono::duration<double, std::milli> (e_end - e_start).count()/1000L;
+      LG << "Epoch [" << ITER << "] Time cost :" << time_taken
```

Comment:
nit: move the space between "cost" and ":". 
"Time cost :" ---> "Time cost: " to be consistent format with "accuracy: "

Reply:
This message seems to be very important and generic to model training. It would be nice to make the message in a standard format across different models and languages.

Reply:
This message seems to be generic to any model training. It would be nice to make the message in a standard format such as a Log message header template across different models and languages.

Reply:
@apeforest i'll fix the alignment. 
with regards to making it a standard for log messages, all the cpp examples need to be revisited, I have created an issue for it https://github.com/apache/incubator-mxnet/issues/11901, I would prefer to do it separately for all CPP Examples.